examples/Building-and-deploying-multi-stage-RecSys/01-Building-Recommender-Systems-with-Merlin.ipynb

```diff
@@ -7,7 +7,7 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    os.environ["TF_GPU_ALLOCATOR"] = "cuda_malloc_async"
</pre><p>guess we need to move this line right after <code>import os</code> to make it effective. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/1022/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
@jperez999 I wanted to say if `os.environ["TF_GPU_ALLOCATOR"] = "cuda_malloc_async"` comes after `import tensorflow` it might work.

Reply:
I think it has to happen in with the `os` call first and `import tensorflow` second so that TF picks up the config from the environment when it loads

Reply:
<p>I say let's move <code>os.environ["TF_GPU_ALLOCATOR"] = "cuda_malloc_async"</code> right after <code>import os</code> based on my experience :)   </p>